src/hotspot/share/gc/parallel/psParallelCompact.cpp

```diff
@@ -30,9 +30,9 @@
 #include "classfile/systemDictionary.hpp"
 #include "code/codeCache.hpp"
 #include "compiler/oopMap.hpp"
+#include "gc/parallel/parMarkBitMap.inline.hpp"
```

Comment:
This uses a case-sensitive sort, whereas I think that when we add includes without using a sorting tool we would do so in a case-insensitive manner. I'm not sure we should make this change here. (The same goes for similar cases in the rest of the patch)